tests/core/kernel-and-base-single-reboot/task.yaml

```diff
@@ -0,0 +1,132 @@
+summary: Exercises a simultaneous kernel and base refresh with a single reboot
+
+# TODO make the test work with ubuntu-core-20
+systems: [ubuntu-core-18-*]
+
+environment:
+    BLOB_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+    snap ack "$TESTSLIB/assertions/testrootorg-store.account-key"
+    #shellcheck source=tests/lib/store.sh
+    . "$TESTSLIB"/store.sh
+
+    setup_fake_store "$BLOB_DIR"
+
+restore: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+    core_snap=core20
+    if os.query is-core18; then
+        core_snap=core18
+    fi
+
+    readlink /snap/pc-kernel/current > pc-kernel.rev
+    readlink "/snap/$core_snap/current" > core.rev
```

Comment:
I'm confused why doing this inside restore?

Reply:
This is fixed now